frontend/src/app/offers/offer-detail/offer-detail.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { OffersService } from '../offers.service';
+import { ActivatedRoute } from '@angular/router';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { OfferDetailComponent } from './offer-detail.component';
+import { Subject } from 'rxjs/Subject';
+
+describe('OfferDetailComponent', () => {
+  let component: OfferDetailComponent;
+  let fixture: ComponentFixture<OfferDetailComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ OfferDetailComponent ],
+      providers: [
+        {
+          provide: OffersService,
```

Comment:
+1

Reply:
No, because this will pull all `OffersService` dependencies (including `HttpClient`, possibly `Router`) and so on. Of course `empty object is temporary before real tests get here, but you should not inject real `OffsersService` here.

Reply:
ok, i missed the `spec` in file name, LOL